benchbuild/utils/compiler.py

```diff
@@ -18,13 +18,21 @@
 Are just convencience methods that can be used when interacting with the
 configured llvm/clang source directories.
 """
+import typing as tp
+
 from plumbum import local
 
 from benchbuild.settings import CFG
 from benchbuild.utils.wrapping import wrap_cc
 
+if tp.TYPE_CHECKING:
+    from mypy_extensions import VarArg
```

Comment:
Codacy found an issue: [Unable to import 'mypy_extensions'](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5694346)